pytorch_lightning/loggers/neptune.py

```diff
@@ -363,3 +357,24 @@ def append_tags(self, tags: Union[str, Iterable[str]]) -> None:
         if str(tags) == tags:
             tags = [tags]  # make it as an iterable is if it is not yet
         self.experiment.append_tags(*tags)
+
+    def _create_or_get_experiment(self):
+        if self.offline_mode:
+            project = neptune.Session(backend=neptune.OfflineBackend()).get_project('dry-run/project')
```

Comment:
this parent folder shall be configurable as the other loggers does, see `save_dir`

Reply:
Hi @Borda, this is not a path, it's actually just a placeholder string. NeptuneLogger doesn't log anything in offline mode.

Reply:
aha, so pls add a comment so we won't ask the same question later...